### PR TITLE
feat(dashboard): Create workflow sheet

### PR DESCRIPTION
**Pull request by desiprisg**
### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

**Review by SokratisVidros — APPROVED** [review]

File: apps/dashboard/src/components/primitives/form/form.tsx [hunk 1]
```diff
@@ -6,8 +6,9 @@ import { Controller, ControllerProps, FieldPath, FieldValues, FormProvider } fro
 import { cn } from '@/utils/ui';
 import { Label } from '@/components/primitives/label';
 import { cva } from 'class-variance-authority';
-import { RiInformationFill } from 'react-icons/ri';
 import { FormFieldContext, FormItemContext, useFormField } from './form-context';
+import { RiErrorWarningFill, RiInformationFill } from 'react-icons/ri';
+import { BsFillInfoCircleFill } from 'react-icons/bs';
```
Comment (BiswaViraj) [inline apps/dashboard/src/components/primitives/form/form.tsx:11]: ❓ **question:** What is our approach for icons for the dashboard?
I see we are also creating react svg files from figma.‏

File: apps/dashboard/src/components/workflow-list.tsx [hunk 2]
```diff
@@ -154,86 +153,71 @@ export const WorkflowList = () => {
...
-                          className="cursor-pointer"
-                          text={workflow.name}
-                          onClick={() => {
-                            navigate(
-                              buildRoute(ROUTES.EDIT_WORKFLOW, {
-                                environmentId: currentEnvironment?._id ?? '',
-                                workflowId: workflow._id,
-                              })
-                            );
-                          }}
```
Comment (BiswaViraj) [inline apps/dashboard/src/components/workflow-list.tsx:178]: i think this is removed by mistake, without this we wont be able to navigate to the edit workflow screen

File: apps/dashboard/src/components/primitives/form/form.tsx (hunk 1 above)
Comment (desiprisg) [reply apps/dashboard/src/components/primitives/form/form.tsx:11]: This was the closest I could find. I generally thought we were using remix icons, but it seems that it's not the case.

File: apps/dashboard/src/components/workflow-list.tsx (hunk 2 above)
Comment (desiprisg) [reply apps/dashboard/src/components/workflow-list.tsx:178]: Maybe I did this on the rebase by accident. I'll fix it.

File: apps/dashboard/src/components/primitives/form/form.tsx (hunk 1 above)
Comment (BiswaViraj) [reply apps/dashboard/src/components/primitives/form/form.tsx:11]: we are also using icons like [this](https://github.com/novuhq/novu/blob/next/apps/dashboard/src/components/icons/notification-5-fill.tsx) by copying from the figma directly

Comment (desiprisg) [reply apps/dashboard/src/components/primitives/form/form.tsx:11]: Yeah but some match icons in `react-icons`, so there's no need to do that. Do you think I should copy the svg in this case?

Comment (BiswaViraj) [reply apps/dashboard/src/components/primitives/form/form.tsx:11]: No i think it's fine, was just curious about how we are using the icons in the new dashboard

**Review by BiswaViraj — APPROVED** [review]

File: apps/dashboard/src/components/primitives/form/form.tsx (hunk 1 above)
Comment (LetItRock) [reply apps/dashboard/src/components/primitives/form/form.tsx:11]: only these are the custom icons: https://www.figma.com/design/iOS9MnuJjLR7avdSIHemFy/novu-v3.0-Design-System-%E2%9A%A1%EF%B8%8F?node-id=125-263&node-type=text&m=dev
the rest are remix icons

**Review by LetItRock — APPROVED** [review]

**Pull request merged**